llvm/tools/spirv-to-ir-wrapper/CMakeLists.txt

```diff
@@ -1,3 +1,9 @@
+
+if (NOT LLVM_TOOL_LLVM_SPIRV_BUILD OR
```

Comment:
I think that this change is not necessary, because `spirv-to-ir-wrapper` does not depend on the translator at build time. Instead it performs a lookup for the binary at runtime:

https://github.com/intel/llvm/blob/52f34fd597fb6d611dd65399c6536f61bc778d69/llvm/tools/spirv-to-ir-wrapper/spirv-to-ir-wrapper.cpp#L63-L69

Reply:
Thanks. The way I've attempted to do this is overly complicated. I should leave spirv-to-ir-wrapper unconditional and just check for the existence of the llvm-spirv target in test/CMakeLists.txt.